### PR TITLE
add https://solid.redpencil.io/ to Pod Providers

### PR DESCRIPTION
**Pull request by ewingson**
d2023-05-22 t04:10

add experimental / production Pod Provider

### CODE REVIEW

File: pages/use-solid.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ The following table lists some Pod Provider:
...
 | [solidweb.me](https://solidweb.me/)                | [Meisdata](https://github.com/serverproject-dev)                           | [Hosteurope](https://www.hosteurope.de)        | EU              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
 | [Data Pod](https://datapod.igrant.io/)                | [iGrant.io, Sweden](https://igrant.io/)                           | [RedPill Linpro](https://www.redpill-linpro.com/en), [AWS](https://aws.amazon.com/), [GCP](https://cloud.google.com/) or [Azure](https://azure.microsoft.com/en-in)        | EU              | [NSS](https://github.com/solid/node-solid-server)        |
+| [redpencil.io](https://solid.redpencil.io/)                | [redpencil.io](https://redpencil.io/)                           | [redhost.be](http://redhost.be/), [Hetzner](https://www.hetzner.com/)        | Finland              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
```
Comment (Vinnl) [inline pages/use-solid.md:46]: redhost.be is giving me a 503 unavailable, and it's not clear to me what having two parties responsible for hosting means here?

(Are you part of the Redpencil team? :) Or if not, can I see somewhere that it's hosted in Finland?)

Comment (ewingson) [reply pages/use-solid.md:46]: Hi Vincent.

a) I do in fact think redhost.be is used for internal administration, that's where our `ssh` goes. we can as well leave that part out, in that case gimme just a call to action and I'll edit the PR.

b) yes, I'm redpencil Junior and you can see it here https://check-host.net/ip-info?host=redpencil.io&csrf_token=4edd32419c19fc696d4c89e2da2b544a5455f650

Comment (Vinnl) [reply pages/use-solid.md:46]: That's great to hear, congrats! And yes, I think it's probably best to just list Hetzner for now. Thanks!

Comment (ewingson) [reply pages/use-solid.md]: yep ;-)

**Review by Vinnl — APPROVED** [review]
Thank you!

**Pull request merged**